deploy/csi-proxy-ds.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: csi-proxy
+  namespace: kube-system # Is this the right namespace?
```

Comment:
Looks good based on https://github.com/kubernetes-csi/csi-driver-smb/blob/master/deploy/csi-smb-node.yaml#L6 and https://github.com/kubernetes-csi/csi-driver-nfs/blob/master/deploy/csi-nfs-node.yaml#L8